src/platform_specific/unix/arc4random.cr

```diff
@@ -3,10 +3,10 @@
 require "c/stdlib"
 
 module Random::System
-  # Fills *buffer* with random bytes using arc4random.
-  #
-  # NOTE: only secure on OpenBSD and CloudABI
   def self.random_bytes(buffer : Bytes) : Nil
+    # Fills *buffer* with random bytes using arc4random.
```

Comment:
Why did you put the doc under the method? (And not above)

Reply:
This is a platform-specific redefinition of a method. It must not override the documentation that is meant to be public. Implementation details shouldn't surface there anyway.